### PR TITLE
More Fixes & Tweaks (12H until Ready to Merge!)

### PR DESCRIPTION
**Pull request by CDAGaming**
Signed-off-by: CDAGaming <cstack2011@yahoo.com>

Self-Explanitory

### CODE REVIEW

File: PoGo.Necrobot.Window/MainClientWindow.xaml.cs [hunk 1]
```diff
@@ -28,7 +28,6 @@
 using PoGo.NecroBot.Logic.Model;
 using DotNetBrowser;
 using DotNetBrowser.WPF;
-using PokemonGo.RocketAPI.Extensions;
```
Comment (Furtif) [inline PoGo.Necrobot.Window/MainClientWindow.xaml.cs]: if build debug missing usings

code 

``
#if DEBUG
                LoggerProvider.Instance.LoggingEnabled = true;
                LoggerProvider.Instance.FileLoggingEnabled = true;
                string logsDir = Path.Combine(appDir, "Logs");
                if (!Directory.Exists(logsDir))
                    Directory.CreateDirectory(logsDir);
                LoggerProvider.Instance.OutputFile = Path.Combine(logsDir, $"DotNetBrowser-{DateTime.UtcNow.ToUnixTime()}.log");
#endif``

Comment (CDAGaming) [inline PoGo.Necrobot.Window/MainClientWindow.xaml.cs]: Oh Ok @Furtif Adding back

Comment (Furtif) [inline PoGo.Necrobot.Window/MainClientWindow.xaml.cs]: lol

Comment (CDAGaming) [inline PoGo.Necrobot.Window/MainClientWindow.xaml.cs]: :P Should be all good. Still WIP as i'm adding more fixes

**Comment by CDAGaming** [comment]
Ready To Merge in 24 Hours

**Comment by CDAGaming** [comment]
PR will be split tomorrow. All that's left is the poll resilts

**Comment by CDAGaming** [comment]
Results*

**Comment by CDAGaming** [comment]
Will Reopen in separated PRs

**Pull request closed without merging**